develop/core/config.py

```diff
@@ -1,5 +1,5 @@
 from django.contrib.sites.models import Site
-from django.utils.translation import ugettext_lazy as _
+from django.utils.translation import gettext_lazy as _
```

Comment:
I was going to mention lots of translation stuff is deprecated in Django 4, but I guess you already checked on that.
https://docs.djangoproject.com/en/4.0/releases/4.0/#features-removed-in-4-0